File: PVP_backend/app/Models/PollutionPoint.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class PollutionPoint extends Model
{
    use HasFactory;

    protected $fillable = [
        'longitude',
        'latitude',
        'rating',
        'type',
        'isActive',
        'reportCount',
    ];
}

File: PVP_android/app/src/main/java/com/natureclean/api/model/PollutionPoint.kt
package com.natureclean.api.model

data class PollutionPoint(
    val id: String? = null,
    val name: String,
    val description: String? = null,
    val longitude: Double? = 0.0,
    val latitude: Double? = 0.0,
    val rating: Int,
    val type: Int,
    val isActive: Int = 1,
    val reportCount: Int = 0,
)
